packages/auth/amplify_auth_cognito_dart/lib/src/auth_plugin_impl.dart

```diff
@@ -279,6 +283,16 @@ class AmplifyAuthCognitoDart extends AuthPluginInterface<
       );
     }
 
+    // Dependencies for AnalyticsMetadataType
+    _stateMachine.addInstance(
+      AmplifySecureStorageDart(
```

Comment:
You're using `AmplifySecureStorageDart` and `AmplifySecureStorageWorker` interchangeably throughout. We should ideally just be using the worker variant

Reply:
Outdated, removed for EndpointInfoStoreManager